src/styles/colors.js

```diff
@@ -1,13 +1,13 @@
 const colors = {
-  colorPrimary: '#3f51b5',
+  colorPrimary: '#A6F3D6',
   colorPrimaryLight: '#757de8',
-  colorPrimaryDark: '#002984',
+  colorPrimaryDark: '#07377A',
   colorSecondary: '#f2f4ff',
   colorSecondaryLight: '#ffffff',
   colorSecondaryDark: '#bfc1cc',
   colorAccent: '#46d3f2',
```

Comment:
Podemos usar branco aqui e usar o colorAccent como background nas telas.

Se preferir pode utilizar `colorAcdent: '#757de8' e colorSecondary: '#FFFFFF'`
```suggestion
  colorAccent: '#FFFFFF',
```

Reply:
Já alterei, obrigada.